recipes/default.rb

```diff
@@ -200,27 +202,23 @@
   action :put
 end
 
-directory librenms_rrddir do
-  owner librenms_username
-  group librenms_group
-  mode '0755'
-  action :create
-  not_if { ::File.exist? librenms_rrddir }
-end
-
-directory librenms_bootstrap_cachedir do
-  owner librenms_username
-  group librenms_group
-  mode '0755'
-  recursive true
-  action :create
-end
+librenms_directories.each do |dir|
+  directory dir do
+    owner librenms_username
+    group librenms_group
+    mode '0775'
+    recursive true
+    action :create
+    not_if { ::File.exist? dir }
+  end
 
-directory librenms_storagedir do
-  owner librenms_username
-  group librenms_group
-  mode '0755'
-  action :create
+  facl dir do
```

Comment:
I don't understand why you have to make sure those acl should hardening and why the directory resource doesn't handle this?

Reply:
I am just following [LibreNMS install guide](https://docs.librenms.org/Installation/Installation-CentOS-7-Apache/), but I guess this is not mandatory for a standard CentOS setup.

Reply:
Sorry I still don't understand the acl issue

Reply:
I have removed it